app/controllers/books_controller.rb

```diff
@@ -42,6 +46,8 @@ def update
     @book = Book.find(params[:id])
 
     if @book.update(book_params)
+      expire_page action: "index"
+
```

Comment:
Just to make sure I understood the code, `expires_page`, removes the page from cache, forcing the server to retrieve it all from the db again, right?

Reply:
Yes